tests/features/cache_tombstone.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Tombstoned cache files are not reported
+
+@skip_below_android_9
+Scenario: A zero-length file is not reported when cache tombstone behavior is set
```

Comment:
Now that we're not checking tombstone behaviour and just removing any zero-length file, this scenario and associated test classes should be renamed to remove references to tombstone behaviour.

Reply:
I've renamed the test scenarios appropriately and have removed code which set the cache behaviour using the `StorageManager`.